### DISCUSSION TITLE
news/20241014/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#399] [Website Discussions]
# news/20241014/

The MKBHD team (aka Marques Brownlee's YouTube team) gives a full masterclass on how they make high quality video.

https://fcp.cafe/news/20241014/

**Comment by Psychilles** [comment]
It warms my heart to see that all the editors are using the magnetic timeline correctly as opposed to Marques himself :P :D. No dis to Marques he's great. But use the F*ing Magnetic Timeline.
